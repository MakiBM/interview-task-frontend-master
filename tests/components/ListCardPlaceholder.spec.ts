import { describe, it, expect, vi } from 'vitest'
import { mount } from '@vue/test-utils'
import Component from '../../src/components/ListCardPlaceholder.vue'

describe('ListCardPlaceholder Snapshot Tests', () => {
  it('matches the snapshot', () => {
    const wrapper = mount(Component, {
      props: {
        title: 'Placeholder Title',
      }
    })

    expect(wrapper.html()).toMatchSnapshot()
  })
})
