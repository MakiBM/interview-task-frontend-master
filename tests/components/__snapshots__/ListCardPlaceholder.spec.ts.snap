// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`ListCardPlaceholder Snapshot Tests > matches the snapshot 1`] = `
"<div class="card h-100 dashed" data-v-5d0701c5="">
  <div class="card-body d-flex align-items-center justify-content-center text-muted" data-v-5d0701c5="">Placeholder Title</div>
</div>"
`;
